assets/authchoice.js

```diff
@@ -28,7 +28,7 @@ jQuery(function($) {
         return this.each(function() {
             var $container = $(this);
 
-            $container.find('a').on('click', function(e) {
+            $container.find('a.auth-link').on('click', function(e) {
```

Comment:
What needs to be adjusted in `clientLink()'?  I'm only seeing the one js file that controls what happens on click.